src/Workspaces/Core/Portable/WorkspacesResources.resx

```diff
@@ -613,4 +613,7 @@
   <data name="Changing_document_property_is_not_supported" xml:space="preserve">
     <value>Changing document properties is not supported</value>
   </data>
+  <data name="Make_field_readonly" xml:space="preserve">
```

Comment:
OK. Is there any particular reason the [Populate Switch message](https://github.com/dotnet/roslyn/blob/master/src/Features/Core/Portable/PopulateSwitch/PopulateSwitchDiagnosticAnalyzer.cs#L19) is at workspace level?